kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/convert/KafkaSchemaRecordType.java

```diff
@@ -0,0 +1,14 @@
+package com.wepay.kafka.connect.bigquery.convert;
+
+
+public enum KafkaSchemaRecordType {
+    VALUE("value"),
+    KEY("key");
+    String str;
+    KafkaSchemaRecordType(String str) {
```

Comment:
I feel like we could verify here that `str` is valid ("value" or "key")

Reply:
We shouldn't need to, this constructor from my understanding is to enable a string value for enum types, and we actually don't use this to construct any new enum types.